open-budget-il/main.py

```diff
@@ -677,22 +680,23 @@ def get_query(self):
         index = search.Index(name="OpenBudget")
         results = []
         try:
-            searchQueryStringList = [queryString]
```

Comment:
I like the way you took it but wouldn't now the url params:
 q=מדינה&year=2014:
will result in this query:
"מדינה AND ((type=BudgetLine AND year=2014))"
?
This means that Entity kinds wouldn't be found in this case. In fact, it's identical to "מדינה AND year=2014"...
Perhaps we should return to your first implementation of using the queryString inside each clause, but keep the fallback in case we can use the simple version.
(this is from a static reading of the code, so I might be wrong)